mkdocs.yml

```diff
@@ -79,3 +80,4 @@ pages:
      - 'Mozilla Grant': 'community/mozilla-grant.md'
      - 'Funding': 'community/funding.md'
      - 'Jobs': 'community/jobs.md'
+     - 'Legacy CoreAPI Docs': 'coreapi/index.md'
```

Comment:
@tomchristie this seems to be the only way I can get `mkdocs` to actually render the page, but I didn't want to add it to the navigation. 

Short of solving #6623, is there a work-around to have the (in the end) 4 CoreAPI related pages rendered, so we can link to them, but NOT part of the nav?

Reply:
I’m not sure there’s a way to do that. Can we gracelessly sidestep the issue by including them on a “docs” page, but at the bottom etc...

Or else including a pre-rendered HTML page in the theme could work.

Reply:
Or just link to the markdown source on GitHib